Revit_Core_Adapter/Revit_Core_Adapter.csproj

```diff
@@ -100,81 +100,41 @@
     <ErrorReport>prompt</ErrorReport>
     <CodeAnalysisRuleSet>ManagedMinimumRules.ruleset</CodeAnalysisRuleSet>
     <TargetFrameworkVersion>v4.7.2</TargetFrameworkVersion>
-    <AssemblyName>Adapter_Revit_UI_2020</AssemblyName>
+    <AssemblyName>Revit_Core_Adapter_2020</AssemblyName>
```

Comment:
May be worth adding Revit 2021 at some point

Reply:
https://github.com/BHoM/Revit_Toolkit/issues/717 there is an issue for that one 👍